Example/TABTestKit_ExampleUITests/BiometricLogin_AuthenticationFailureTests.swift

```diff
@@ -11,7 +11,7 @@ import TABTestKit
 final class BiometricLogin_AuthenticationFailureTests: TABTestCase, SystemPreferencesContext {
     
     override func preLaunchSetup(_ launch: @escaping () -> Void) {
-        resetAllPrivacyPrompts()
+        resetAllPrivacyPrompts().execute()
```

Comment:
I'm not really a fan of having to call `execute` all over the place :/

Reply:
I get that, the problem is that I need to return a description. Suggestions are welcome.